### PR TITLE
staging-repos: add note that these are to be imported as k8s/*

### PR DESCRIPTION
**Pull request by sttts**
Fixes https://github.com/kubernetes/sample-controller/issues/17
Fixes https://github.com/kubernetes/sample-controller/issues/20

### CODE REVIEW

File: staging/src/k8s.io/sample-apiserver/README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Demonstration of how to use the k8s.io/apiserver library to build a functional API server.
 
+**Note:** go-get or vendor this package as `k8s.io/sample-apiserver`.
```
Comment (nikhita) [inline staging/src/k8s.io/sample-apiserver/README.md]: not sure how this fixes the sample-controller issues? isn't this for sample-apiserver?

Comment (sttts) [reply staging/src/k8s.io/sample-apiserver/README.md]: oops. Yes :) But, we should add that to all staging repos, if we don't say that already.

Comment (nikhita) [reply staging/src/k8s.io/sample-apiserver/README.md]: > But, we should add that to all staging repos, if we don't say that already.

:+1:

Comment (sttts) [reply staging/src/k8s.io/sample-apiserver/README.md]: Added a note to all repos (other than client-go which has the README outside of staging).

**Pull request merged**